templates/zerver/integrations.html

```diff
@@ -1677,6 +1683,48 @@
       <img class="screenshot" src="/static/images/integrations/svn/001.png" />
     </div>
 
+    <div id="teamcity" class="integration-instructions">
+
+      <p>See your Teamcity build status in Zulip!</p>
+
+      <p>First, create the stream you'd like to use for Teamcity
+      notifications, and subscribe all interested parties to this stream. We
+      recommend the name <code>teamcity</code>.</p>
+
+      <p>Next, set up a bot for the integration. You'll need the bot's API key
+      to construct a URL for Teamcity.</p>
+
+      <p>Next, install the <a href="https://github.com/tcplugins/tcWebHooks/releases">tcWebHooks plugin</a>
+      onto your Teamcity server. Follow the plugin instructions in your
+      Teamcity documentation, or refer to
+      <a href="https://confluence.jetbrains.com/display/TCD9/Installing+Additional+Plugins">the online Teamcity documentation</a>.</p>
+
+      <p>Next, in your Teamcity project overview page,
+      click the "Webhooks" tab, and add a new project webhook.
+      Enter the following URL:</p>
+
+      <p><code>{{ external_api_uri }}/v1/external/teamcity?api_key=abcdefgh&amp;stream=teamcity</code></p>
+
+      <p>where <code>api_key</code> is the API key of your Zulip bot,
+      and <code>stream</code> is the stream name you want the
+      notifications sent to.</p>
+
+      <p>Uncheck all "Trigger on Events" options, and check
+      "Trigger when build is Successful" and "Trigger when build Fails".
+      Optionally, check "Only trigger when build changes from Failure to Success"
+      and "Only trigger when build changes from Success to Failure".</p>
+
+      <p>Set the Payload Format to "JSON" and save your webhook.</p>
+
+      <img class="screenshot" src="/static/images/integrations/teamcity/001.png" />
+
+      <p><b>Congratulations! You're done!</b><br /> Your Teamcity build
+       notifications will appear in Zulip:</p>
+      <img class="screenshot" src="/static/images/integrations/teamcity/002.png" />
```

Comment:
Normally we do this screenshot with a complete recipient header so one can see what topics the integration uses -- can you retake this with that changed?